src/GafferScene/TweakPlug.cpp

```diff
@@ -240,11 +279,127 @@ class NumericTweak
 			case TweakPlug::Multiply :
 				data->writable() = sourceDataCast->readable() * tweakDataCast->readable();
 				break;
+			case TweakPlug::Min :
+				/// \todo is this neccessary?
+				assert( sourceDataCast->readable().dimensions() == tweakDataCast->readable().dimensions() );
+
+				for( unsigned int i = 0; i < sourceDataCast->readable().dimensions(); ++i )
+				{
+					data->writable()[i] = std::min( sourceDataCast->readable()[i], tweakDataCast->readable()[i] );
+				}
+				break;
+			case TweakPlug::Max :
+				/// \todo is this neccessary?
```

Comment:
As above

Reply:
Fixed in 28eadea4221e3faf6fe4d8ffad2b0591d78fc7fe